pkg/addons/clean_test.go

```diff
@@ -0,0 +1,222 @@
+package addons
+
+import (
+	"reflect"
+	"testing"
+
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+)
+
+func TestClean(t *testing.T) {
+	tests := []struct {
```

Comment:
it must be possible to do something nicer? this will grow into misery.
testdata directory, subdirectory per test, %03d-input.yaml, %03d-expected.yaml, loop, unmarshal, clean, compare?

Reply:
better?

Reply:
not really - take a look at https://github.com/jim-minter/openshift-azure/commit/tests for what I had in mind